test/navbar-components/navigationBarTest.js

```diff
@@ -377,15 +377,18 @@ describe('navigationBar tests', function () {
         .waitUntil(() =>
           this.app.client
             .activateURLMode()
-            .waitForExist(urlbarIcon)
-            .getAttribute(urlbarIcon, 'class').then((classes) =>
+            .waitForExist(urlBarIcon)
+
+            // TODO: Fix
+            .getAttribute(urlBarIcon, 'class').then((classes) =>
               classes.includes('fa-unlock') && classes.includes('insecure-color')
```

Comment:
@cezaraugusto as I don't add any change on source code on this PR, would you mind please fixing the failing tests? thanks!